utils/utils.py

```diff
@@ -7,9 +7,6 @@
 import warnings
 import collections
 import fiona
-import matplotlib
```

Comment:
Why was matplotlib taken out?

Reply:
Not sure why I got rid of it, but I changed it back for now.